src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -96,4 +109,18 @@ public double getRightWheelsRPM() {
   public double getAverageWheelsRPM() {
     return (getLeftWheelsRPM() + getRightWheelsRPM()) / 2;
   }
+
+  @Override
+  public double getMeasurement(){
+    return m_leftFront.getSelectedSensorPosition();
```

Comment:
Should return the average measure distance by all the encoders. This method was already added by @VenusM3, `getDistance()`